index.js

```diff
@@ -1,5 +1,11 @@
 'use strict';
 
 module.exports = {
-  name: '@ember/ordered-set'
+  name: '@ember/ordered-set',
+
+  options: {
+    babel: {
+      loose: true,
```

Comment:
is that public API? how will this work with the delayed transpilation that Kelly is working on?

Reply:
@Turbo87 - Yes, it is public API from ember-cli-babel's perspective (see [here](https://github.com/babel/ember-cli-babel#options)).

RE: delayed transpilation, this addon would _not_ get to opt-out of "per-addon" transpilation (e.g. it isn't eligible for delayed transpilation) because it includes a custom plugin that _may_ not be present in the host app.  I think that this is perfectly fine (both short term and longer term), but I am also thinking that we may want to make `ember-compatibility-helpers` a part of `ember-cli-babel` out of the box (so everyone gets these capabilities for free) and if we did do that then we would use delayed transpilation again...

Reply:
> I am also thinking that we may want to make ember-compatibility-helpers a part of ember-cli-babel out of the box

sounds good to me 👍